src/System.Private.ServiceModel/tests/Benchmarks/PerfUsingCoreWCFService/CoreWCFPerfService/MyCustomValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using CoreWCF.IdentityModel.Selectors;
+using System;
+using System.Threading.Tasks;
+
+namespace WCFCorePerfService
+{
+    public class MyCustomValidator : UserNamePasswordValidator
+    {
+        public override ValueTask ValidateAsync(string userName, string password)
+        {            
+            if (!string.IsNullOrEmpty(userName) && !string.IsNullOrEmpty(password))
+            {
+                return new ValueTask(Task.CompletedTask);
+            }
```

Comment:
You can just return `default` as the default ValueTask represents a completed operation.